### PR TITLE
OPSEXP-703: add checkov test and fixes for initial list

### PR DESCRIPTION
**Pull request by sergiuv2020**

### CODE REVIEW

**Review by nilsSure — APPROVED** [review]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -198,6 +198,7 @@ _helm_deploy: &helm_deploy
     --set controller.service.annotations."service\.beta\.kubernetes\.io/aws-load-balancer-ssl-negotiation-policy"="ELBSecurityPolicy-TLS-1-2-2017-01" \
     --set controller.service.annotations."service\.beta\.kubernetes\.io/aws-load-balancer-security-groups"="${AWS_SG}" \
     --set controller.publishService.enabled=true \
+    --set controller.admissionWebhooks.enabled=false \
```
Comment (gavincornwell) [inline .travis.yml:201]: If we're adding this to our build should we update the documentation too?

File: helm/alfresco-content-services/6.1.N_values.yaml [hunk 2]
```diff
@@ -20,10 +20,27 @@ repository:
     pullPolicy: Always
     internalPort: 8080
     hazelcastPort: 5701
-  initContainer:
-    image:
-      repository: busybox
-      tag: latest
+  initContainers:
```
Comment (gavincornwell) [inline helm/alfresco-content-services/6.1.N_values.yaml:23]: Nice, you've taken the opportunity to externalise the init containers as well.

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -7,5 +7,7 @@ repos:
       - id: helm-lint
       - id: kubepug-minimum
       - id: kubepug-recommended
-      # We will re-enable this via OPSEXP-703
-      # - id: checkov-helm
\ No newline at end of file
+  - repo: https://github.com/Alfresco/acs-deployment
+    rev: 5c9347de2a3ec7e3281f1b5e8da71e395f79c2a6
```
Comment (gavincornwell) [inline .pre-commit-config.yaml:11]: Are there any implications to using a specific revision here?

File: helm/alfresco-content-services/templates/deployment-repository.yaml [hunk 4]
```diff
@@ -171,7 +177,7 @@ spec:
             - name: ca-cert
               mountPath: /var/run/secrets/cacert
         - name: set-perms
-          image: busybox
+          image: busybox:1
```
Comment (gavincornwell) [inline helm/alfresco-content-services/templates/deployment-repository.yaml:180]: Shouldn't we externalise this one?

File: helm/alfresco-content-services/values.yaml [hunk 5]
```diff
@@ -21,10 +21,27 @@ repository:
...
     hazelcastPort: 5701
-  initContainer:
-    image:
-      repository: busybox
-      tag: latest
+  initContainers:
+    db:
+      image:
+        repository: busybox
+        tag: 1
```
Comment (gavincornwell) [inline helm/alfresco-content-services/values.yaml:28]: It looks like we've got two different approaches in the charts, sometimes we split the repository and tag and sometimes not i.e. "busybox:1"

**Review by gavincornwell — CHANGES_REQUESTED** [review]
On the whole looks great, just a couple of minor inline comments

File: .pre-commit-config.yaml (hunk 3 above)
Comment (sergiuv2020) [reply .pre-commit-config.yaml:11]: a revision is mandatory, however it cannot be the master branch. I know it looks ugly, didnt find a way around it for now, however on the next release of the chart we will be able to use the tag.

File: helm/alfresco-content-services/values.yaml (hunk 5 above)
Comment (sergiuv2020) [reply helm/alfresco-content-services/values.yaml:28]: yes, fixed the other occurences also, we will have repository:tag everywhere

File: helm/alfresco-content-services/templates/deployment-repository.yaml (hunk 4 above)
Comment (sergiuv2020) [reply helm/alfresco-content-services/templates/deployment-repository.yaml:180]: done, missed it

**Review by gavincornwell — APPROVED** [review]
LGTM

**Pull request merged**